File: AspNetMvcUnleashed/0672329980_source/Chapter11/VB/MvcApplication1/MvcApplication1/Controllers/CacheControlController.vb
Public Class CacheControlController
    Inherits Controller

    Public Function Index() As String
        Response.Cache.SetCacheability(HttpCacheability.Private)
        Response.Cache.SetMaxAge(TimeSpan.FromSeconds(10))
        Return DateTime.Now.ToString("T") & " <a href='/CacheControl/Index'>link</a>"
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter11/CS/MvcApplication1/MvcApplication1/Controllers/CacheControlController.cs
using System;
using System.Web;
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class CacheControlController : Controller
    {
        public string Index()
        {
            Response.Cache.SetCacheability(HttpCacheability.Private);
            Response.Cache.SetMaxAge(TimeSpan.FromSeconds(10));
            return DateTime.Now.ToString("T") + " <a href='/CacheControl/Index'>link</a>";
        }

    }
}
